extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/exec/WorkerContext.java

```diff
@@ -73,4 +74,9 @@
   DruidNode selfNode();
 
   Bouncer processorBouncer();
+
+  default File tempDir(int stageNumber, String id)
```

Comment:
Nit brigade!  I think we tend to use `tmp` in other parts of the code?

Reply:
I was modeling it off `File tempDir()` that is already here without args. I can change both? This one doesn't matter much to me either way. Thoughts?

Reply:
I like consistency.  If it's always `tmp` then it's much easier to say that next time it should be `tmp` as well.